### PR TITLE
updated developers guide

### PR DESCRIPTION
**Pull request by KevinLohJunYong**

### CODE REVIEW

**Review by nicholastanvis — CHANGES_REQUESTED** [review]
you made a whole uml diagram but you didn't include it in the developer guide

**Comment by nicholastanvis** [comment]
> you made a whole uml diagram but you didn't include it in the developer guide

please generate the images from your diagram and put it into the docs/images folder

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+The following activity diagram summarizes what happens when a 
+user executes the cdelete command:
+
+![Delete Contact Command Activity Diagram](images/DeleteContactCommandActivityDiagram.png)
+
+The following sequence diagram summarizes what happens when a
+user executes the cdelete command:
+
+![Delete Contact Command Sequence Diagram](images/DeleteContactCommandSequenceDiagram.png)
+###[Proposed] Enquire if time interval is free
```
Comment (nicholastanvis) [inline docs/DeveloperGuide.md]: add a space after the triple hashtags

**Review by nicholastanvis — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
 # Implementation
 This section describes some noteworthy details on how certain features are implemented.
 
-### [Proposed] Enquire if time interval is free
+### Deleting a contact in the address book. 
+
+The command immediately responsible for this is the 'DeleteContactCommand'
+
+An outline of the proposed implementation is as follows:
```
Comment (tsh22) [inline docs/DeveloperGuide.md]: Remove proposed

**Review by tsh22 — CHANGES_REQUESTED** [review]

**Review by nicholastanvis — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1,6 +1,27 @@
...
+The command immediately responsible for this is the 'DeleteContactCommand'
+
+An outline of the implementation is as follows:
+
+The `TeachingAssistantParser` should accept the command 
+word `cdelete`. It will then get the `DeleteContactCommandParser`
+to parse and return the `DeleteContactCommand`. Subsequently, it
+returns `CommandResult` which is shown on the command result
+field of the GUI as the information of the user that is deleted.
+`
```
Comment (tsh22) [inline docs/DeveloperGuide.md]: Rogue ` punctuation here

**Review by tsh22 — CHANGES_REQUESTED** [review]

**Review by tsh22 — APPROVED** [review]

**Review by goatygoatygoat — APPROVED** [review]

**Pull request merged**